bench-vortex/benches/random_access.rs

```diff
@@ -27,6 +27,7 @@ fn random_access(c: &mut Criterion) {
     });
 
     let dataset = BenchmarkDatasets::PBI(Medicare1);
+    dataset.write_as_parquet();
```

Comment:
this needs to happen to convert the CSVs to parquet, so that the next line can convert the parquet's to lance